.docker/Dockerfile-build

```diff
@@ -32,6 +30,13 @@ RUN mkdir -p /var/lib/sqlite
 RUN chown ory:ory /var/lib/sqlite
 VOLUME /var/lib/sqlite
 
+# Exposing the ory home directory to simplify passing in Kratos configuration (e.g. if the file $HOME/.kratos.yaml 
+# exists, it will be automatically used as the configuration file).
+VOLUME /home/ory
```

Comment:
Would it make sense to document that somewhere?

Reply:
Yes, definitely. I first wanted to get the blessing on the general changes. I think it makes sense to add a section on the Dockerfiles and how to work with them. 

The documentation should cover: 
* different types of images (with / without SQLite)
* configuration management (mout config file, custom image)
* inject "secrets" via ENV)

Reply:
Sounds good, this is good to merge from my side - should I go ahead?

Reply:
Yes. So Documentation will follow in another PR, ok?

Reply:
yup!